doc/mmp_testcases.txt

```diff
@@ -0,0 +1,66 @@
+Poor Man's Multiple Mount Protection Test Cases
+
+Variables to Consider
+* Host ID
+* Activity Present
+* VDEV state
+* zpool import -f
+* Tunable zfs_mmp_interval
+
```

Comment:
I ended up adding another tunable, zfs_mmp_fail_intervals.  This describes how many zfs_mmp_interval periods can go by without a successful mmp write, before we suspend the pool.  We should talk with Brian to decide whether/how to test that.